common/subset-tests.js

```diff
@@ -0,0 +1,27 @@
+// Only test a subset of tests with, e.g., ?1-10 in the URL
```

Comment:
Maybe add that this works well with the  `<meta name="variant" ... >` mechanism ?